reader.go

```diff
@@ -51,8 +51,11 @@ func (sfr *FactsetReader) Read(fRes factsetResource, dest string) (string, error
 	if err != nil {
 		return lastVers, err
 	}
+	factsetFiles := strings.Split(fRes.fileNames, ";")
+	for _, factsetFile := range factsetFiles {
+		err = sfr.unzip(lastVers, factsetFile, dest)
```

Comment:
Only the value of `err` for the last file will be used, others will be masked.  This should probably return on the first non-nil error.